tests/unittests/Foundation/ReferenceFoundation/TestNSString.mm

```diff
@@ -595,19 +594,19 @@ guard ensureFiles([@"/tmp/ABC/temp.txt"]) else {
     {
         NSString* path = @"/tmp/";
         auto result = path.stringByDeletingLastPathComponent;
-        ASSERT_OBJCEQ(result, static_cast<NSString*>(_CFGetSlashStr()));
+        ASSERT_OBJCEQ(result, @"/");
```

Comment:
Additionally, I believe in the lost previous iterations where this was ifdef'd, you wanted this to not be ifdef'd? What is the exact concern/scenario you have/are worried about?

---
In reply to: [72336763](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72336763) [](ancestors = 72336763,72325600)